### PR TITLE
Release 27.9

### PR DESCRIPTION
**Pull request by orndorffgrant**
TODO:
- [x] clean up extra commits from rebase
- [x] bump version
- [x] write changelog entry
- [x] Create LP bug and reference in changelog

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,54 @@
...
+  * New upstream release 27.9 (LP: #1973099)
+    - cli:
+      + for json formatted output, include additional_info for some errors
+    - daemon:
+      + replace ua-license-check timer with ubuntu-advantage.service daemon
+      + detects on-boot if pro license was added and runs auto-attach
+      + only runs on gcp and does not continuously long-poll by default for now
+    - enable:
+      + fix error message on wrong service name when unattached
+      + treat an empty list affordance as un-affordable
```
Comment (lucasmoura) [inline debian/changelog:29]: I think we can drop this since it is not user specific

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+    - refactor python to avoid instantiating UAConfig extra times
+    - refactor python to always use messages module for strings
+    - rm the old marker file that triggered ua-license-check.path
+    - remove unnecessary deb-systemd-helper check in ua-messaging cleanup
+    - clean up old ua-license-check state
+    - run new cloud-id-shim script
+  * d/tools/postrm
+    - clean up ubuntu-advantage-daemon log files
+  * New upstream release 27.9 (LP: #1973099)
+    - cli:
```
Comment (lucasmoura) [inline debian/changelog:21]: I think we can advertise that we have a new command that updates MOTD and APT messages. Even though they are already on the refresh section

**Review by renanrodrigo — APPROVED** [review]
+1

**Comment by lucasmoura** [comment]
@orndorffgrant I think we have some lintian errors regarding the changelog that we need to address:
```
W: ubuntu-advantage-tools: debian-changelog-line-too-long line 35                                                                                                                                                  
W: ubuntu-advantage-tools: debian-changelog-line-too-long line 43                                                                                                                                                  
W: ubuntu-advantage-tools: debian-changelog-line-too-long line 44                                                                                                                                                  
W: ubuntu-advantage-tools: debian-changelog-line-too-long line 45                                                                                                                                                  
W: ubuntu-advantage-tools: debian-changelog-line-too-long line 48 
```

**Comment by orndorffgrant** [comment]
@lucasmoura I think I've addressed everything

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,56 @@
...
+      + cache "apt-cache policy" output to avoid unnecessary subp calls
+    - proxy:
+      + apt_http(s)_proxy renamed to global_apt_http(s)_proxy
+      + apt_http(s)_proxy config var names will still work
+      + new ua_apt_http(s)_proxy for only ua-related apt traffic (LP: #1956764)
+      + global_apt_http(s)_proxy and ua_apt_http(s)_proxy cannot be set at the
+        same time
+    - realtime: adjust warning to clarify that a manual revert is possible
+    - refresh:
+      + new subcommand `ua refresh messages` to update motd and apt messages
```
[the remark this answers is not in the document]
Comment (orndorffgrant) [reply debian/changelog]: @lucasmoura changed and pushed!

**Review by renanrodrigo — APPROVED** [review]

**Pull request merged**